server/package.json

```diff
@@ -7,7 +7,7 @@
 		"start": "node app.js",
 		"test": "echo \"Error: no test specified\" && exit 0",
 		"lint": "eslint \"./**/*.js\"",
-		"dev": "NODE_ENV=development nodemon app.js",
+		"dev": " SET NODE_ENV=development & nodemon app.js",
```

Comment:
I suppose we don't need to commit that since it may break the command for people working on Unix

Reply:
For a windows machine you can run 

`npm install -g win-node-env`

to fix this issue and leave the configuration as it is to *nix machines.